slack-plugin/src/main/kotlin/slack/gradle/SlackProperties.kt

```diff
@@ -253,6 +253,13 @@ public class SlackProperties private constructor(private val project: Project) {
   public val kotlinDaemonArgs: String
     get() = stringProperty(KOTLIN_DAEMON_ARGS_KEY, defaultValue = "")
 
+  /**
+   * Flag to enable the new K2 compiler. Plumbed into the kotlinOptions.useK2 property and here to
+   * allow for use from a `local.properties` file.
+   */
+  public val useK2: Boolean
+    get() = booleanProperty("slack.useK2", defaultValue = false)
```

Comment:
Nice. It looks like there’s a nice performance boost with this new compiler from https://kotlinlang.org/docs/whatsnew17.html#new-kotlin-k2-compiler-for-the-jvm-in-alpha